profileFunctions.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
```

Comment:
All queries can be called `$query` since they are inside their own function scope.

i.e. no need for `$textQuery` , `$emailQuery`, etc.

Reply:
fixed